src/components/Petitions/index.js

```diff
@@ -8,35 +8,29 @@ import PetitionsFilters from 'components/PetitionsFilters';
 import NoResults from 'components/NoResults';
 import Pagination from 'containers/Pagination';
 
-const Petitions = ({
-  petitions,
-  isLoading,
-  title,
-  autocompleteProps
-}) => (
+const Petitions = (props) => (
```

Comment:
Ah… I remember now :) I need to pass all the props to the children PetitionsFilters component now, so I didn’t destructure the props object.